test/src/main/java/org/apache/accumulo/test/functional/FateConcurrencyIT.java

```diff
@@ -204,6 +207,11 @@ public void getFateStatus() {
     Instance instance = connector.getInstance();
     String tableId;
 
+    // development testing - force multiple compactions to see that it's handled.
+    if (runMultipleCompactions) {
+      runMultipleCompactions();
+    }
+
```

Comment:
If it is a more reliable test when running more than one compaction, I say just have it always do that.

Reply:
Nevermind, I see what you are doing... maybe just update the comment to make it more clear.  Maybe use the word "debug" somewhere so its clear this is just to help debug a test failure.